exercises/practice/matching-brackets/README.md

```diff
@@ -0,0 +1,20 @@
+# Matching Brackets
```

Comment:
Could you remove this file? It'll be automatically created and downloaded when using the CLI.

Reply:
done!